pyproject.toml

```diff
@@ -62,7 +62,8 @@ test = [
     "async_generator >=1.10,<2.0.0",
     "python-multipart >=0.0.5,<0.0.6",
     "aiofiles >=0.5.0,<0.6.0",
-    "flask >=1.1.2,<2.0.0"
+    "flask >=1.1.2,<2.0.0",
+    "anyio >=3.1.0,<4",
```

Comment:
It's an `starlette` requirement, so I guess we don't need it here?

Reply:
this depends on `anyio[trio]` now, which isn't a starlette requirement